crates/solidity/outputs/cargo/build/src/main.rs

```diff
@@ -17,7 +17,8 @@ use solidity_language::GrammarConstructorDslV1;
 fn main() -> Result<()> {
     // Generate files in the source crate:
     {
-        let grammar = Grammar::from_dsl_v1();
+        let grammar = Grammar::from_dsl_v2();
+
```

Comment:
looks like both parser tests codegen and sanctuary tests are still using DSL V0. I wonder why didn't we see failures already.
I will investigate.